packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,17 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClients(config: JWTConfig): JwksClient[] {
+  return config.wellKnownUrls.map((url) =>
+    jwksClient({
+      cache: true,
+      rateLimit: true,
```

Comment:
The case should be the second mentioned.
In our case rate limiter comes into play if the cached key has expired or is not found in the cache, the impact of rate limiting is minimized when caching is active since the cached keys will often handle key lookups without needing to hit the rate limiter, but anyway if the cache is empty or expired and multiple requests for JWKS are made before the cache is repopulated, the rate limiting mechanism will throttle the external calls to prevent overwhelming the JWKS server.